src/app/chat/view.nim

```diff
@@ -67,6 +90,33 @@ QtObject:
     read = getActiveChannelIdx
     write = setActiveChannelByIndex
     notify = activeChannelChanged
+  
+  proc activeStickerPackChanged*(self: ChatsView) {.signal.}
+
+  proc setActiveStickerPackById*(self: ChatsView, id: int) {.slot.} =
+    if self.activeStickerPackId == id:
+      return
+
+    self.activeStickerPackId = id
+    self.activeStickerPackChanged()
+
+  proc getActiveStickerPackId(self: ChatsView): QVariant {.slot.} =
+    newQVariant(self.activeStickerPackId)
+
+  QtProperty[QVariant] activeStickerPack:
+    read = getActiveStickerPackId
+    write = setActiveStickerPackByIndex
+    notify = activeStickerPackChanged
```

Comment:
we only need to set the active sticker pack from QML, but the full QtProperty variant bells and whistles aren't necessary so all of the above can be replaced with

```nim
proc activeStickerPackChanged*(self: ChatsView) {.signal.}

proc setActiveStickerPackById*(self: ChatsView, id: int) {.slot.} =
  self.activeStickerPackId = id
  self.activeStickerPackChanged()
```

Reply:
Fair enough, definitely not needed. 

I kept the early return as we don't need to signal that the stickers were updated to QML if the sticker pack id wasn't changed.